torchvision/models/detection/backbone_utils.py

```diff
@@ -25,13 +25,13 @@ class BackboneWithFPN(nn.Module):
     Attributes:
         out_channels (int): the number of channels in the FPN
     """
-    def __init__(self, backbone, return_layers, in_channels_list, out_channels):
+    def __init__(self, backbone, return_layers, in_channels_list, out_channels, extra_blocks=LastLevelMaxPool()):
```

Comment:
nit: can you instead do
```python
def __init__(self, backbone, return_layers, in_channels_list, out_channels, extra_blocks=None):
    if extra_blocks is None:
        extra_blocks = LastLevelMaxPool()
```
This avoids the common python gotcha of passing a mutable default argument to a function

Reply:
Sure thing. It shouldn't be an issue in this case, since we shouldn't mutate `extra_blocks` I believe, but I see where you're coming from.